agent/scripts/init.rhel

```diff
@@ -223,14 +236,18 @@ EOF
 
   if running ; then
     if [ -z "${NR_SILENT}" -a -z "${SILENT}" ]; then
-      success
-      echo
+      if [ $has_initd_functions -eq 1 ]; then
+        success
```

Comment:
There is an argument to be had on whether we should log success using our own functions if we do not have access to initd functions. I am leaning towards no. The reason being: [initd functions](https://github.com/fedora-sysv/initscripts/blob/main/etc/rc.d/init.d/functions#L561) have pre-checks and pos-executions that happen that our internal functions would be unable to replicate.

Reply:
I don't think this can stay as is. The init script must send some information back about it succeeding or failing. This is the behavior when `/etc/init.d/functions` file is present:
```
[root@centos9 ~]# /sbin/service newrelic-daemon start
Starting newrelic-daemon:                                  [  OK  ]
[root@centos9 ~]# /sbin/service newrelic-daemon stop
Stopping newrelic-daemon:                                  [  OK  ]
```

This is the behavior when `/etc/init.d/functions` file is not present:
```
[root@centos9 ~]# /sbin/service newrelic-daemon start
Starting newrelic-daemon: [root@centos9 ~]# /sbin/service newrelic-daemon stop
Stopping New Relic Daemon (newrelic-daemon) ... [root@centos9 ~]#
```

Notice that when `/etc/init.d/functions` file is not present, shell prompt appears on the same line as `Starting newrelic-daemon:` message.

Reply:
fixed in [f078212](https://github.com/newrelic/newrelic-php-agent/pull/919/commits/f07821222788bed4ee2230986f8d8e03733986b3)